generate_outputs.py

```diff
@@ -103,6 +111,156 @@
+                    labels = deepcopy(td[field])
+                    for l in labels:
+                        l["SchemeID"] = target_scheme.scheme_id
+                    remapped[field] = labels
+                td.append_data(remapped, Metadata(user, Metadata.get_call_location(), TimeUtils.utc_now_as_iso_string()))
+
+            # 4. Convert the fields in the bungoma/kilifi/kiambu coding plans relevant to analysis to their unified
+            # fields. This is mostly like folding messages -> individuals, except we now have to account for some fields
+            # being None, which is why there are some special case strategies implemented inline here.
+            def assert_equal(x, y):
+                if x is None:
+                    return y
+                if y is None:
+                    return x
+
+                assert x == y, f"{x} != {y}"
+                return x
+
+
+            def demog_labels(code_scheme, x, y):
+                if code_scheme.get_code_with_code_id(x["CodeID"]).control_code == "NA":
+                    return y
+                if code_scheme.get_code_with_code_id(y["CodeID"]).control_code == "NA":
+                    return x
+
+                assert x["CodeID"] == y["CodeID"]
+                return x
+
+            remappings = {
+                "rqa_s01e01_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E01),
+                "rqa_s01e02_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E02),
+                "rqa_s01e03_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E03),
+                "rqa_s01e04_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E04),
+                "rqa_s01e05_coded": partial(FoldStrategies.list_of_labels, CodeSchemes.ALL_LOCATIONS_S01E05),
+
+                "rqa_s01e01_raw": FoldStrategies.concatenate,
+                "rqa_s01e02_raw": FoldStrategies.concatenate,
+                "rqa_s01e03_raw": FoldStrategies.concatenate,
+                "rqa_s01e04_raw": FoldStrategies.concatenate,
+                "rqa_s01e05_raw": FoldStrategies.concatenate,
+
+                "gender_coded": partial(demog_labels, CodeSchemes.GENDER),
+                "age_coded": partial(demog_labels, CodeSchemes.AGE),
+                "age_category_coded": partial(demog_labels, CodeSchemes.AGE_CATEGORY),
+                "county_coded": partial(demog_labels, CodeSchemes.KENYA_COUNTY),
+                "constituency_coded": partial(demog_labels, CodeSchemes.KENYA_CONSTITUENCY),
+                "disabled_coded": partial(demog_labels, CodeSchemes.DISABLED),
+
+                "gender_raw": assert_equal,
+                "age_raw": assert_equal,
+                "location_raw": assert_equal,
+                "disabled_raw": assert_equal,
+
+                "baseline_community_awareness_coded": partial(demog_labels, CodeSchemes.ALL_LOCATIONS_BASELINE_COMMUNITY_AWARENESS),
```

Comment:
I presume this had multiple labels too  so we we FoldStrategies.list_of_labels and handle them as rqa as requested by Anna. This can be fixed her or in a future PR.

Reply:
Yes, I'll open another PR today that has this change